src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -149,12 +149,16 @@ private function registerAnnotationLoaders(ContainerBuilder $container)
     private function registerFileLoaders(ContainerBuilder $container)
     {
         $paths = [];
+        $globOptions = GLOB_BRACE | GLOB_NOSORT;
```

Comment:
Would go for a dependency on `symfony/finder` in the bundle then.

>but we don't need to reinvent the well.

Having it in Extension or as a proper class doesn't make any difference on that side: we are re-doing, in a much simpler way obviously, what the Finder already do. My suggestion was just about moving that bit out of the Extension to keep it only for file registration and not polluting it with finding the files.